src/surrogates/src/InterpolationSurrogateIOASCII.C

```diff
@@ -59,7 +59,7 @@ namespace QUESO
 
     // Only processor 0 does the reading.
     // We'll broadcast the data as needed
-    if( env.fullRank() == root )
+    if( env.fullRank() == (int)root )
```

Comment:
@roystgnr I'd like to know what you think about changes like this.  Another option is to have our `MpiComm` and `BaseEnvironment` store an `unsigned int` member variable instead, so that casts like this aren't needed.